kafka/pykafka/cluster.py

```diff
@@ -95,6 +97,27 @@ def _update_topics(self, metadata):
             else:
                 self._topics[name].update(meta)
 
+    def discover_offset_manager(self, consumer_group_name):
```

Comment:
This function is in `Cluster` since it needs access to all brokers so it can figure out which to return. It's possible that it would be better for it to live elsewhere, but this is my best guess at the moment.